sky/serve/control_plane.py

```diff
@@ -120,21 +143,22 @@ def terminate(request: fastapi.Request):
     _infra_provider = infra_providers.SkyPilotInfraProvider(
         args.task_yaml,
         args.service_name,
-        readiness_path=service_spec.readiness_path,
-        readiness_timeout=service_spec.readiness_timeout,
+        readiness_suffix=service_spec.readiness_suffix,
+        initial_delay_seconds=service_spec.initial_delay_seconds,
         post_data=service_spec.post_data)
 
     # ======= Autoscaler =========
     _autoscaler = autoscalers.RequestRateAutoscaler(
         _infra_provider,
         frequency=20,
-        min_nodes=service_spec.min_replica,
-        max_nodes=service_spec.max_replica,
+        min_nodes=service_spec.min_replicas,
+        max_nodes=service_spec.max_replicas,
         upper_threshold=service_spec.qps_upper_threshold,
         lower_threshold=service_spec.qps_lower_threshold,
         cooldown=60,
```

Comment:
As we discussed, we want to enable faster scaling. say scale up X replicas at once etc. We can do it in next PR.